libvast/vast/system/actors.hpp

```diff
@@ -271,7 +277,8 @@ using index_actor = typed_actor_fwd<
   // the nil uuid.
   // TODO: Add options to do an in-place transform keeping the old ids,
   // and to make a new partition preserving the old one(s).
-  caf::replies_to<atom::apply, transform_ptr, uuid>::with<vast::uuid>,
+  caf::replies_to<atom::apply, transform_ptr,
+                  uuid>::with<partition_synopsis_pair>,
```

Comment:
You could avoid a bit of boilerplate with:
```suggestion
  caf::replies_to<atom::apply, transform_ptr,
                  uuid>::with<uuid, partition_synopsis_ptr>,
```
It's a matter of style, but in this case I think there is no ambiguity.

Reply:
This won't work with CAF 0.18, which removes this automatic tuple wrapping and unwrapping. Please don't.